### ISSUE TITLE
Extracting the media name from an edl

### ISSUE THREAD

**Issue by frenchie1980** [PixarAnimationStudios/OpenTimelineIO#600]
How do I extract the Media string from an edl like this?

```
000001  A002C001_190710_R6U6             V     C        01:13:32:14 01:13:33:24 01:00:00:00 01:00:01:10
000002  A001C012_190710_R6U6             V     C        01:03:25:18 01:03:27:07 01:00:01:10 01:00:02:24
000003  A001C006_190710_R6U6             V     C        00:45:46:15 00:45:47:24 01:00:02:24 01:00:04:08
```

All I want is the unresolved media clip name (eg `A002C001_190710_R6U6`)

But all I can seem to get is the clip name and the cut range info, and a missing reference object. I don't want the media resolved to any real path. I just want the media name as it exists in the edl in text form

```
Clip("000001", MissingReference(None, None, {}), TimeRange(RationalTime(110314, 25.0), RationalTime(35, 25.0)), {})
Clip("000002", MissingReference(None, None, {}), TimeRange(RationalTime(95143, 25.0), RationalTime(39, 25.0)), {})
Clip("000003", MissingReference(None, None, {}), TimeRange(RationalTime(68665, 25.0), RationalTime(34, 25.0)), {})
```

The code I'm using curently is:

```
edl = "path to my file"
timeline = otio.adapters.read_from_file(edl, rate=25)
for clip in timeline.each_clip():
    print clip
```

**Comment by reinecke** [comment]
This is a known issue that the adapter currently doesn't parse tape name. This is pretty high up in my queue to fix. Would you, by chance, be able to provide a sample EDL file that I could check the fixes against?

**Comment by reinecke** [comment]
Oh, correction. This is actually a different case.
Can you try this:
```
edl = "path to my file"
timeline = otio.adapters.read_from_file(edl, rate=25)
for clip in timeline.each_clip():
    print(clip.metadata['cmx_3600']['reel'])
```

**Comment by frenchie1980** [comment]
Yes that worked. I was on v0.9, which didn't show the metadata. v0.11 does

thanks

**Comment by jminor** [comment]
Addressed by #392
